appcues/src/main/java/com/appcues/debugger/ui/ToastView.kt

```diff
@@ -0,0 +1,157 @@
+package com.appcues.debugger.ui
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.appcues.R.string
+import com.appcues.debugger.model.DebuggerToast.ScreenCaptureFailure
+import com.appcues.debugger.model.DebuggerToast.ScreenCaptureSuccess
+import com.appcues.ui.theme.AppcuesColors
+import kotlinx.coroutines.delay
+
+private const val TOAST_LENGTH = 3_000L
+
+@Composable
+internal fun BoxScope.ToastView(debuggerState: MutableDebuggerState) {
+
+    when (val toastState = debuggerState.toast.targetState) {
+        is ScreenCaptureSuccess -> SuccessToast(toast = toastState, debuggerState = debuggerState)
+        is ScreenCaptureFailure -> FailureToast(toast = toastState, debuggerState = debuggerState)
```

Comment:
There is some commonality in these toasts (general size and position) but enough difference (color, one has button, behaviors) that I found it simplest to just split out into two separate compositions that we can manage independently, IMO.